e2e/Makefile

```diff
@@ -31,7 +31,7 @@ renderTestCase:
 	@D="$$(mktemp -d)" ;\
 	mkdir -p "$$D/case" ;\
 	$(BIN_DIR)/envsubst -i "$$TPL" -o "$$D/case/00-case.yaml" ;\
-	echo -n "$$D"
+	echo "$$D\c"
```

Comment:
this works on my mac but I see a literal \c on my linux VM. Could we instead do `printf "$$D\n"`? That seems consistent on both